dotcom-rendering/src/web/components/Kicker.tsx

```diff
@@ -7,6 +8,7 @@ type Props = {
 	color: string;
 	showPulsingDot?: boolean;
 	showSlash?: boolean;
+	showLineBreak?: boolean;
```

Comment:
I completely get this is a pattern so the consistency is good, just a thought.

`showSlash` and `showLinebreak` set as default `true` feels confusing as I would assume `undefined` is analogous to `false` as it the convention with other HTML props e.g. [`disabled`](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes).

> The values "true" and "false" are not allowed on boolean attributes. To represent a false value, the attribute has to be omitted altogether.

e.g.
```ts
// I would expect this not to show the line break nor slash
<Kicker />

// As an example, I would expect this render differently to the above
<Kicker showSlash={true} />
```

So would `hideSlash` and `hideLineBreak` make more sense?

Reply:
Yep this is a totally fair comment! I was following the logic of showSlash here but you're right, it is a little confusing. Given that showSlash is going to be removed shortly, I won't refactor this prop but I will with `linebreak`